recipes/zip/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+add_executable(${PROJECT_NAME} test_package.c)
+target_link_libraries(${PROJECT_NAME} ${CONAN_LIBS})
+
```

It's also nice to test the official targets from upstream

```suggestion

find_package(zip REQUIRED)

add_executable(${PROJECT_NAME} test_package.c)
target_link_libraries(${PROJECT_NAME} zip::zip)
```

---

same:
well, not sure if it improves anything, doesn't seem to be required (builds just fine without this line), but only makes code longer -> harder to read and understand